docs/configuration/testing-behavior.rst

```diff
@@ -1,10 +1,10 @@
-Configuration
-================
+.. _testing-behavior:
 
-Cosmos offers a few different configuration options for how your dbt project is run and structured. This page describes the available options and how to configure them.
+Testing Behavior
+================
 
-Testing
-----------------------
+Testing Configuration
+---------------------
 
 By default, Cosmos will add a test after each model. This can be overridden using the ``test_behavior`` field. The options are:
```

Comment:
It is probably worth stating that the default behavior of Cosmos is different from the default behaviour from dbt (running each test individually) and explain why.